customers/tests/conftest.py

```diff
@@ -43,23 +43,25 @@ def hki_profile_address() -> dict:
     return MOCK_HKI_PROFILE_ADDRESS
 
 
+def get_customer_profile_dict() -> dict:
+    faker = Faker()
+    profile = CustomerProfileFactory()
+    return {
+        "id": to_global_id(ProfileNode, profile.id),
+        "first_name": profile.user.first_name,
+        "last_name": profile.user.last_name,
+        "primary_email": {"email": profile.user.email},
+        "primary_phone": {"phone": faker.phone_number()},
+        "primary_address": MOCK_HKI_PROFILE_ADDRESS,
+    }
```

Comment:
I had added this on a previous PR since I needed the functionality, just double-checking that we won't have repeated utils.